### PR TITLE
Update cards database

### PR DESCRIPTION
**Pull request by lichmanuyk**

### CODE REVIEW

File: client/src/store/cards/cards.reducer.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export const cardsReducer = (
     case CardsActionTypes.LoadCardsError:
       return {
         ...state,
+        cards: state.cards,
```
Comment (DrJekill352) [inline client/src/store/cards/cards.reducer.ts]: Why did you add this line?

Comment (vharadkou) [inline client/src/store/cards/cards.reducer.ts]: unnecessary, spread with state (...state) do the same

Comment (lichmanuyk) [reply client/src/store/cards/cards.reducer.ts]: Fixed

Comment (lichmanuyk) [reply client/src/store/cards/cards.reducer.ts]: Fixed
